packages/components/Icon/src/legacy/Icon.tsx

```diff
@@ -68,19 +69,24 @@ function renderFontIcon(iconProps: IconProps) {
 function renderSvg(iconProps: IconProps) {
   const svgIconProps: SvgIconProps = iconProps.svgSource;
   const { accessible, accessibilityLabel, width, height, color } = iconProps;
-  const viewBox = iconProps.svgSource.viewBox;
   const style = mergeStyles(iconProps.style, rasterImageStyleCache({ width, height }, [width, height])[0]);
 
+  let svgProps: SvgProps | UriProps = { width, height, color };
```

Comment:
Sorry, I didn't realize that `SvgProps` and UriProps` are actually two different types. I think what was earlier is probably better 😅

Reply:
No worries, just reverted it!